invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/tombstone.html

```diff
@@ -8,17 +8,46 @@
 
 {%- set title = _("Tombstone") + " | " + config.THEME_SITENAME -%}
 
-{% block message %}
-  <h1><i class="fa fa-flash"></i> {{_('Gone')}}</h1>
+{%- block message %}
+  <h1><i class="fa fa-flash"></i> {{ _('Gone') }}</h1>
   <p>
-  {% trans sitename=config.THEME_SITENAME %}
-    The record  you are trying to access was removed from {{sitename}}. The
+  {%- trans sitename=config.THEME_SITENAME %}
+    The record  you are trying to access was removed from {{ sitename }}. The
     metadata of the record is kept for archival purposes.
-  {% endtrans %}
+  {%- endtrans %}
   </p>
-  {% if record and record.removal_reason %}
-  <p>
-    <strong>{{ _("Reason for removal:") }}</strong> {{record.removal_reason}}
-  </p>
-  {% endif %}
+  <div class="ui segment">
+  {%- if record and record.tombstone and record.tombstone.is_visible %}
```

Comment:
good point!

i've updated the check in the tombstone page, since it was not logically equivalent to the negation of this check here though.
the main difference was the case if there _is_ a (deleted) record that _does not have_ tombstone information.
previously, this would cause the `tombstone.html` template to be rendered, but only the the "heading" part.
now, this case will result in a 404.

note, however:
with the normal flow, a record should never get into the "deleted" status without having tombstone information attached.
so maybe you would actually prefer me to remove the checks for `if record and record.tombstone`, as they potentially mask deeper issues?